doc/devdocs/log-files.md

```diff
@@ -0,0 +1,15 @@
+# Log files
+
+To read PowerToys log files you can use the Analogy Log Viewer with the DLL from @LiorBanai.
+
+> I have created log parser for the logs created by PowerToys.
+> It may be relevant/usefull for troubleshooting issues and errors.
+> 
+> I have multi purpose log application at [Analogy Log Viewer](https://github.com/Analogy-LogViewer/Analogy.LogViewer)
+> A also created dedicated Parser at [Analogy.LogViewer.PowerToys](https://github.com/Analogy-LogViewer/Analogy.LogViewer.PowerToys)
+> 
+> for now, It targeted .net core 3.1 so just need to download [Analogy Log Viewer latest](https://github.com/Analogy-LogViewer/Analogy.LogViewer/releases/tag/V4.2.9), download [PowerToy Parser dll](https://github.com/Analogy-LogViewer/Analogy.LogViewer.PowerToys/releases/tag/V0.1.0), put it at the same folder of the application and run it.
+> 
+> ![image](https://user-images.githubusercontent.com/36262995/94992149-a533d880-0590-11eb-9440-e5b8e608ff10.png)
```

Comment:
Alt text for image

Reply:
It's only a information picture that isn't relevant

Reply:
Ok